doc/source/language/user_functions.rst

```diff
@@ -349,24 +468,22 @@ declaring them first.  Most such languages have provided a way to
 catch the problem, and allow you to instruct the compiler "please
 don't let me do that.  Please force me to declare everything".
 
-The way that is done in kerboscript is by using a ``NOLAZYGLOBAL`` 
-section, :ref:`as described here <nolazyglobal>`.
+The way that is done in kerboscript is by using a ``@LAZYGLOBAL`` 
+compiler directive, :ref:`as described here <lazyglobal>`.
 
-Had the function above been wrapped inside a NOLAZYGLOBAL section,
+Had the function above been wrapped inside a ``@LAZYGLOBAL off.`` section,
```

Comment:
good catch.  I thought I found all of those and changed them.  I'll issue a PR to change it.